ios/keymanios.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```

Comment:
Is this needed for ios ? Just a new artefact of XCode 9.3 maybe?

Reply:
Yes, it's a new artefact of XCode. It basically tells XCode that it's already done the check to see if the app is 32-bit in order to warn the developer that the clock is ticking on conversion of all code to 64-bit.